.buildkite/pipelines/security_solution_quality_gate/mki_periodic/mki_periodic_defend_workflows.yml

```diff
@@ -17,6 +17,8 @@ steps:
         parallelism: 5
         retry:
           automatic:
+            - exit_status: "-1"
```

Comment:
These aren't spot instances, historically we've very rarely seen standard instances drop.  Only when there's a buildkite / gcp outage.

Reply:
@MadameSheema how would you like to proceed?

Reply:
(only this file, the others seem to be)

Reply:
Ah, understood.